internal/core/application/wallet_service.go

```diff
@@ -699,10 +701,20 @@ type accountLastDerivedIndex struct {
 }
 
 func (a *accountLastDerivedIndex) total() int {
-	return a.external + a.internal
+	return (a.external + 1) + (a.internal + 1)
```

Comment:
external and internal can take values in the range [-1, N] where -1 means no used addresses and N (variable value) stands for the total num of used addresses found.

Without this fix, I had the daemon panicking for a wallet that had 1 external used address and no internal used. This situation caused the total() method to incorrectly return -1. This fix the problem.